react/components/SliderTrack.tsx

```diff
@@ -22,6 +26,35 @@ const SliderTrack: FC = ({ children }) => {
     return index >= currentSlide && index < currentSlide + slidesToShow
   }
 
+  if (isSSR) {
+    const slideWidthPercentage = 100 / slidesPerPage
+
+    return (
+      <Fragment>
+        {childrenArray.slice(0, slidesPerPage).map((child, index) => (
+          <div
+            key={index}
```

Comment:
This could be harmful in the reconciliation process when a list is not static. Keep using the `index` as `key` if you are sure that it will never be reordered, filtered, get new data, etc.

Reply:
Awesome catch! And you're right about the `index` possible hurting React's reconciliation algorithm.
However, that's not the case in this component. That's because this whole `<Fragment>` is wrapped around an `if`  statement that will evaluate to `true` while on SSR, but will evaluate to `false` while this is rendered to client-side, and a whole new tree will be returned, so no matter what, the reconciliation algorithm would either not be executed or simply rebuild the tree anyway.

Reply:
Cool then :)